index.html

```diff
@@ -5,7 +5,6 @@
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css" />
   </head>
-
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
     <h1 id="title">HEADING 1</h1>
```

Comment:
- [ ] I congratulate your for the professionalism shown when giving the comments 👏. Your comments are more detailed and well explained. However, we need to ensure that a student understands every comment independently. I suggest that you ensure that the comments are used on specific lines of codes instead of putting all the comments on one section of the code. 
For example: For the comment that says 10-32, it would be much better if you make the inline comment for that specific section instead of including others like 1-10.